lib/unparser/buffer.rb

```diff
@@ -77,7 +76,6 @@ def unindent
     #
     def nl
       write(NL)
-      self
```

Comment:
btw, I had to do this because of a bug in `mutant`'s `--since` functionality where if you delete code then the next neighboring method will get mutated (and `#write` was not mutation covered)

Reply:
yes, this is a known bug in mutants git integration. usually I just fix the adjacent mutations, but its time I fix it anyway.